synology_dsm/api/core/upgrade.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+"""DSM Upgrade data and actions."""
+
+
+class SynoCoreUpgrade(object):
+    """Class containing upgrade data and actions."""
+
+    API_BASE_KEY = "SYNO.Core.Upgrade"
+    API_SERVER_KEY = API_BASE_KEY + ".Server"

```

Comment:
```suggestion
    API_KEY = "SYNO.Core.Upgrade"
    API_SERVER_KEY = API_KEY + ".Server"
```

We always use `API_KEY`

Reply:
fixed